customer-address.go

```diff
@@ -88,6 +88,26 @@ func (r customerAddressRepository) Update(id int64, address shopify.CustomerAddr
 	return responseDTO.ToShopify(), nil
 }
 
+func (r customerAddressRepository) SetDefault(id int64, addressID int64) (shopify.CustomerAddress, error) {
+	url := r.createURL(fmt.Sprintf("customers/%v/addresses/%v/default.json", id, addressID))
```

Comment:
See you don't even have to convert it here...

Reply:
Yeah, it does simplify a bit our parsing, on the other hand we are not following their API to the letter 😄